03-seeking-help.Rmd

```diff
@@ -124,6 +124,11 @@ your issue.
 > What is the difference between the `sep` and `collapse` arguments?
 > 
 
+> ## Challenge 3 {.challenge}
+> Use help to find out which function and with what parameters should you use
```

Comment:
Change: "and with what parameters should you use" -> "and with what parameters you could use"